src/core/session.js

```diff
@@ -98,13 +98,13 @@ export class Session {
     }
   }
 
-  refresh(url, requestId) {
+  refresh = debounce((url, requestId) => {
     const isRecentRequest = requestId && this.recentRequests.has(requestId)
     if (!isRecentRequest) {
       this.cache.exemptPageFromPreview()
       this.visit(url, { action: "replace" })
     }
-  }
+  }, 150)
```

Comment:
Maybe something like `Turbo.session.pageRefreshDelay`? The problem with setting such a high number, say, 250 ms, is that the sense of realtime is lost. If you get a page refresh, wait 300 ms, and the page takes 250 ms to load, that's 550 ms. That's why I didn't want to leave such a high number either.

Reply:
That's a good point. With `Turbo.session.pageRefreshDelay` would I be able to configure it per page? For example, I know that a given page is slow to refresh, so the `Turbo.session.pageRefreshDelay` is 300ms there, but everything else is 150ms. This would give the developer flexibility per page.

Reply:
Ah, in that case, if the configuration is per page, I think it would be better if the setting is done via a meta tag, such as `turbo-page-refresh-delay`. If we set this up in the session, it has a tinge of something more fixed or permanent than something variable per page. What do you think? I would also like to get opinions from maintainers before going forward with something.

Reply:
I like the idea of using an option as @ArturT suggests, similar to `progressBarDelay`. What about `Turbo.session.pageRefreshDebouncePeriod`?

Reply:
Great! What default value do you think is right?

Reply:
I think 150 is fine, we can go with that for now. Regarding whether it's too low for certain requests, notice this is meant to debounce the stream actions that arrive via web sockets, so it's independent of page rendering times.